app/controllers/sufia/admin_sets_controller.rb

```diff
@@ -1,7 +1,6 @@
 module Sufia
   class AdminSetsController < ApplicationController
     include CurationConcerns::CollectionsControllerBehavior
-    layout "sufia-one-column"
```

Comment:
@mjgiarlo any controller that includes on inherits from CurationConcerns::CollectionsControllerBehavior gets `with_themed_layout '1_column'` by default.  No need to override or set again.